spec-insert/spec/_fixtures/input/example_code.md

```diff
@@ -0,0 +1,19 @@
+<!-- spec_insert_start
```

Comment:
Add a test case with `body` and one with boolean request param without value (ex. `<endpoint>?pretty`)